.github/CODEOWNERS

```diff
@@ -33,6 +33,8 @@ Tasks/AzureContainerAppsV0/    @cormacpayne    @daniv-msft
 
 Tasks/AzureContainerAppsV1/    @cormacpayne    @daniv-msft
 
+Tasks/AzureDevOpsTestPlanV0/     @rasunkar  @microsoft\adoautotest
```

Comment:
Seeing an 'invalid owner' error here. Is this a concern?

Reply:
Copied it from here:
![image](https://github.com/microsoft/azure-pipelines-tasks/assets/92331194/9a17abbc-570e-420a-9d38-98a3a13de8e6)

Reply:
Sure. As there is a prior example, this should be good